precompiled-token/src/getACAinfo.ts

```diff
@@ -0,0 +1,67 @@
+import { use } from 'chai';
+import { Contract } from 'ethers';
+import ADDRESS from "@acala-network/contracts/utils/Address"
+
+import { evmChai } from '@acala-network/bodhi';
+
+const Token = require('@acala-network/contracts/build/contracts/Token.json');
```

Comment:
should be also use `import` to be more consistent?

Reply:
Sure, I don't have the preference and could find arguments for both (being concise vs. showing different options), It seems that you have a preference, so I changed this :ok_hand: